notebooks/266-table-question-answering/266-table-question-answering.ipynb

```diff
@@ -0,0 +1,413 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #4.    result = compiled_model((inputs["input_ids"], inputs["attention_mask"], inputs["token_type_ids"]))
</pre><p>does input has some additional fields. I think, it maybe more user friendly to use dict:</p><p> </p><p>result = compiled_model(dict(inputs))</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1408/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Reply:
Replaced